src/index.js

```diff
@@ -1,59 +1,57 @@
 import React from 'react';
 import PropTypes from 'prop-types';
-import Payment from 'payment';
+import creditCardType, { types as cardTypes } from 'credit-card-type';
+import luhn from 'luhn';
+
+import { dankort, laser, visaElectron } from './cardTypes';
+import { sanitizeNumber } from './utils';
 
 class ReactCreditCards extends React.Component {
   constructor(props) {
     super(props);
 
-    this.setCards();
+    creditCardType.updateCard(cardTypes.MAESTRO, {
+      patterns: [
+        493698,
+        [5000, 5018],
+        [502000, 506698],
+        [506779, 508999],
+        [56, 59],
+        63,
+        67,
+        6,
+      ],
+    });
+    creditCardType.updateCard(cardTypes.HIPERCARD, {
+      patterns: [
+        384100,
+        384140,
+        384160,
+        606282,
+        637095,
+        637568,
+      ],
+    });
+    creditCardType.addCard(dankort);
+    creditCardType.addCard(laser);
+    creditCardType.addCard(visaElectron);
```

Comment:
not super happy with all these configurations being done here but couldn't think of a better place, open to suggestions